src/AppTokenRetrival.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+
+import { Mount } from 'react-lifecycle-components'
+import { useLocation } from 'react-router'
+
+import { apolloClient } from './client/apolloClient'
+import { StorageContainer } from './utils/StorageContainer'
+
+export const AppTokenRetrival = ({ children }) => {
+  const location = useLocation()
+
+  return (
+    <StorageContainer>
+      {(storageState) => (
+        <Mount
+          on={() => {
+            if (location.hash.includes('token')) {
+              var params = {}
```

Comment:
`let`*

Reply:
But most preferrably `Array#reduce` + `const`

Reply:
haha muskelminne, in Swift a let is immutable but and a var is mutable 😭